sandbox/main.qml

```diff
@@ -164,6 +164,11 @@ StatusWindow {
                             selected: page.sourceComponent == othersComponent
                             onClicked: page.sourceComponent = othersComponent
                         }
+                        StatusNavigationListItem {
+                            title: "Settings"
```

Comment:
Can we call this `StatusExpandableSettingsItem`? The idea is, that eventually every component gets its own page (I know we have pages like "Others" and "Buttons" but we gotta start somewhere :D)